sessionize/app/src/main/java/co/touchlab/sessionize/event/EventFragment.kt

```diff
@@ -104,26 +104,28 @@ class EventFragment : Fragment() {
     }
 
     private fun updateContent(event: SessionInfo, formattedRoomTime:String) {
-        val adapter = EventDetailAdapter(activity!!)
+        if(::recycler.isInitialized) {
```

Comment:
Does this imply this is getting called before or after the view is initialized? Are we not shutting down the viewmodel properly?